src/modules/chat_deleted_messages/index.js

```diff
@@ -57,24 +57,34 @@ class ChatDeletedMessagesModule {
 
   handleDelete(name, targetMessageId) {
     const deletedMessages = settings.get(SettingIds.DELETED_MESSAGES);
-    const showDeletedMessages = deletedMessages === DeletedMessageTypes.SHOW;
-    const hideDeletedMessages = deletedMessages === DeletedMessageTypes.HIDE;
-    if (!hideDeletedMessages && !showDeletedMessages) {
+    if (
+      deletedMessages !== DeletedMessageTypes.HIDE &&
+      deletedMessages !== DeletedMessageTypes.SHOW &&
+      deletedMessages !== DeletedMessageTypes.HIGHLIGHT
+    ) {
       return false;
     }
     const messages = findAllUserMessages(name, targetMessageId);
     messages.forEach((message) => {
       const $message = $(message);
-      if (hideDeletedMessages) {
-        $message.hide();
-      } else if (showDeletedMessages) {
-        $message.toggleClass(CHAT_LINE_DELETED_CLASS, true);
-        /* eslint-disable-next-line func-names */
-        $message.find(CHAT_LINE_LINK_SELECTOR).each(function () {
-          const $link = $(this);
-          $link.removeAttr('href');
-        });
-        $message.find(CHAT_LINE_CLIP_CARD_SELECTOR).remove();
+      switch (deletedMessages) {
+        case DeletedMessageTypes.HIDE:
+          $message.hide();
+          break;
+        case DeletedMessageTypes.HIGHLIGHT:
+          $message.css('background-color', '#dc2626');
+        // eslint-disable-next-line no-fallthrough
```

Comment:
nit: why not this? to avoid using an eslint-ignore

```js
case DeletedMessageTypes.HIGHLIGHT:
case DeletedMessageTypes.SHOW:
   if (deletedMessages === DeletedMessageTypes.HIGHLIGHT) {
      $message.css('background-color', '#dc2626');
   }
   ...

Reply:
Good call, change made. Thanks!